packages/core/src/diff/changes/input.ts

```diff
@@ -50,21 +50,25 @@ export function inputFieldRemoved(
 }
 
 export function buildInputFieldAddedMessage(args: InputFieldAddedChange['meta']) {
-  return `Input field '${args.addedInputFieldName}' of type '${args.addedInputFieldType}' was added to input object type '${args.inputName}'`;
+  return `Input field '${args.addedInputFieldName}' of type '${args.addedInputFieldType}'${args.addedFieldDefault ? ` with default value '${args.addedFieldDefault}'` : ''} was added to input object type '${args.inputName}'`;
 }
 
 export function inputFieldAddedFromMeta(args: InputFieldAddedChange) {
   return {
     type: ChangeType.InputFieldAdded,
-    criticality: args.meta.isAddedInputFieldTypeNullable
+    criticality: args.meta.addedToNewType
       ? {
-          level: CriticalityLevel.Dangerous,
+          level: CriticalityLevel.NonBreaking,
         }
-      : {
-          level: CriticalityLevel.Breaking,
-          reason:
-            'Adding a required input field to an existing input object type is a breaking change because it will cause existing uses of this input object type to error.',
-        },
+      : args.meta.isAddedInputFieldTypeNullable || args.meta.addedFieldDefault !== undefined
```

Comment:
Should be make `args.meta.addedFieldDefault !== undefined` actually safe?

Reply:
Adjusted to be nonBreaking.